src/opm/input/eclipse/Schedule/Well/WellInjectionProperties.cpp

```diff
@@ -232,7 +241,7 @@ namespace Opm {
             if (switching_from_prediction ||
                 switching_from_BHP_control ||
                 switching_from_producer) {
-                this->resetDefaultHistoricalBHPLimit();
```

Comment:
Is this function still used?  if not, we can remove this function and copy the comments to where the special (689*.*) value is set.

Reply:
Yes, it's used in restart code. To make matters worse these values are not stored in eclipse restart files, so even there the behavior is kinda funky.